xbmc/LangInfo.cpp

```diff
@@ -300,6 +302,7 @@ void CLangInfo::CRegion::SetGlobalLocale()
     strLocale = lcl.name();
     current_locale = current_locale.combine< std::collate<wchar_t> >(lcl);
     current_locale = current_locale.combine< std::ctype<wchar_t> >(lcl);
+    current_locale = current_locale.combine< std::time<wchar_t> >(lcl);
```

Comment:
As I already wrote above, I used ```std::time_get``` and ```std::time_put``` instead of ```std::time``` for testing un Linux. It compiled, but it doesn't have the expected change.